main.go

```diff
@@ -50,8 +51,9 @@ var (
 
 	persistenceStoragePath = flag.String("storage.local.path", "/tmp/metrics", "Base path for metrics storage.")
 
-	remoteTSDBUrl     = flag.String("storage.remote.url", "", "The URL of the OpenTSDB instance to send samples to.")
-	remoteTSDBTimeout = flag.Duration("storage.remote.timeout", 30*time.Second, "The timeout to use when sending samples to OpenTSDB.")
+	remoteStorageType    = flag.String("storage.remote.type", "opentsdb", "The type of remote storage to use. Valid values: 'opentsdb', 'influxdb'.")
+	remoteStorageURL     = flag.String("storage.remote.url", "", "The URL of the remote storage instance to send samples to.")
```

Comment:
This assumes that all future storage will have a URL. But others might need other parameters (let's say a mysql-style DSN) or even more than only one parameter (location of a credential file, other config parameters). How about replacing "storage.remote.type" and "storage.remote.url" by "storage.remote.opentsdb-url" and "storage.remote.influxdb-url"? We could later add "storage.remote.hipsterdb-dsn", "storage.remote.hipsterdb-extent-size" and such.

If more than one of these flags is set, we could either bail out or even send samples to all of them... (the latter is simpler for now).